lib/nuget-parser/index.ts

```diff
@@ -181,28 +181,13 @@ Will attempt to build dependency graph anyway, but the operation might fail.`);
     );
   }
 
-  // Ensure `dotnet` is installed on the system or fail trying.
-  const version = await dotnet.validate();
```

Comment:
Can you change it to just `await dotnet.validate()`. Still want to verify they have `dotnet` installed before continuing.

Reply:
sure thing